packages/@aws-cdk/aws-route53-targets/lib/cloudfront-target.ts

```diff
@@ -11,12 +12,36 @@ export class CloudFrontTarget implements route53.IAliasRecordTarget {
    */
   public static readonly CLOUDFRONT_ZONE_ID = 'Z2FDTNDATAQYW2';
 
+  /**
+   * Get the hosted zone id for the current scope.
+   *
+   * @param stack - Stack in which the resource is defined
+   */
+  public static getHostedZoneId(stack: Stack) {
```

Comment:
does this method need to be public?
I think it'd make sense to add a constant similar to the `CLOUDFRONT_ZONE_ID` constant if we want to expose the hosted zone id for `aws-cn`

Reply:
@shivlaks given that both `CloudFrontTarget` and `UserPoolDomainTarget` rely on a hosted zone id and the existing `CLOUDFRONT_ZONE_ID` property already being public I was thinking that is probably best to keep this behaviour for users to ensure the least interruption. Ideally I would like to remove the existing `CLOUDFRONT_ZONE_ID` attribute but that would mean a breaking change so I kept it for now.